cpp/source/deal.II/mp_values.templates.h

```diff
@@ -130,7 +130,8 @@ PorousElectrodeMPValues<dim, spacedim>::PorousElectrodeMPValues(
   double const pores_characteristic_dimension = to_meters(matrix_phase_database->get<double>("pores_characteristic_dimension"));
   double const pores_geometry_factor          = matrix_phase_database->get<double>("pores_geometry_factor");
   double const mass_density                   = to_kilograms_per_cubic_meter(matrix_phase_database->get<double>("mass_density"));
-  double const electrical_conductivity        = 1.0/to_ohm_meter(matrix_phase_database->get<double>("electrical_resistivity"));
+  double const electrical_resistivity         = to_ohm_meter(matrix_phase_database->get<double>("electrical_resistivity"));
+  double const electrical_conductivity        = (electrical_resistivity>1e300) ? 0. : 1.0/electrical_resistivity;
```

Comment:
OK I get now [where](https://github.com/ORNL-CEES/Cap/blob/master/cpp/source/deal.II/mp_values.h#L127) we get the high resistivity from